wpeverest/everest-forms
# Fix - Radio, checkbox, select issues

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes design issue of Radio button, checkbox, Select option for Twenty Twenty-One Theme.
![image](https://user-images.githubusercontent.com/66351929/116077148-def73880-a6b4-11eb-949a-e4304387374a.png)

This is the images after solved the issue
![image](https://user-images.githubusercontent.com/66351929/116076844-7f009200-a6b4-11eb-97e9-0cfffcce578d.png)


### How to test the changes in this Pull Request:

1. Checkout to this branch
2. Activate Twenty Twenty-One Theme 
3. Please check if the design issue still exist or not for Radio button, Select option, Checkbox

### Types of changes:




### Other information:




### Changelog entry

> Fix - Radio, checkbox, select issues
